app/lib/settings/scoped_settings.rb

```diff
@@ -47,7 +47,6 @@ def []=(key, value)
       record.update!(value: value)
 
       Rails.cache.write(Setting.cache_key(key, @object), value)
-      value
```

Comment:
`Rails.cache.write` does not have a documented return value. I would need some reasoning for leaving  this function without a return value.

Reply:
ok, the fact this value is written and used as the return was lost on me.. It seems to alert in rubocop as "Variable `value` used in void context".

https://rubocop.readthedocs.io/en/latest/cops_lint/#lintvoid